### PR TITLE
Krona

### PR DESCRIPTION
**Pull request by nekrut**
Krona tool for visualization of taxonomy datasets in galaxy. Next I will work on converting kraken data into richer galaxy taxonomy format.

### CODE REVIEW

File: tools/krona/krona.xml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+            <param name="combine_inputs" value="False"/>
+            <output name="output" file="krona_test1.html" ftype="html" lines_diff="30"/>
+        </test>
+    </tests>
+    <help>
+        <![CDATA[
+
+**What it does**
+
+Creates an ineractive Krona_ chart based on Galaxy taxonomic representations.
```
Comment (bgruening) [inline tools/krona/krona.xml]: Why `Galaxy` here?

**Comment by bgruening** [comment]
LGTM!
@nekrut join the IUC with your tools!

File: tools/krona/tool_dependencies.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<tool_dependency>
+	<package name="perl" version="5.18.1">
+        <repository name="package_perl_5_18" owner="iuc" prior_installation_required="True" />
+    </package>
+
```
Comment (bgruening) [inline tools/krona/tool_dependencies.xml]: How does this relates to https://github.com/galaxyproject/tools-iuc/pull/232?

Comment (nekrut) [inline tools/krona/tool_dependencies.xml]: directly. it depends on it

File: tools/krona/krona.xml (hunk 1 above)
Comment (nekrut) [inline tools/krona/krona.xml]: Not sure I understand. Krona has a tool `ktImportGalaxy` while turns galaxy taxonomy datatype into krona html document.

File: tools/krona/tool_dependencies.xml (hunk 2 above)
Comment (bgruening) [inline tools/krona/tool_dependencies.xml]: But you haven’t defined a dependency here, you are installing it by your own.
You also have not defined a requirement in your tool, afaik.

Comment (nekrut) [inline tools/krona/tool_dependencies.xml]: Perhaps a part of the confusion is the decision to put deps into tools-uic and actual tools into tools-devteam. Seems a bit too complicated for me though

File: tools/krona/tool_dependencies.xml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<tool_dependency>
+	<package name="perl" version="5.18.1">
```
Comment (bgruening) [inline tools/krona/tool_dependencies.xml]: Indentation is wrong here.

File: tools/krona/tool_dependencies.xml (hunk 2 above)
Comment (bgruening) [inline tools/krona/tool_dependencies.xml]: :)
You need to include something like:
```
<package name="krona" version="2.5">
        <repository name="package_krona_2_5" owner="iuc" />
</package>
```
The perl requirement can go away, imho.
To stop the confusion commit your tools to IUC as well :)

File: tools/krona/krona.xml (hunk 1 above)
Comment (bgruening) [inline tools/krona/krona.xml]: Not sure this detail is interesting for a user. I would argue that a user don't care about the underlying representation details.
Leave it like it is, it's fine with me.

File: tools/krona/tool_dependencies.xml (hunk 3 above)
Comment (davebx) [inline tools/krona/tool_dependencies.xml]: I'll fix that in post, too.

**Comment by nsoranzo** [comment]
This is mostly overlapping with https://toolshed.g2.bx.psu.edu/view/crs4/taxonomy_krona_chart which is available since 2014-04-11. I think updating the old one would better serve the 75 users that have already installed it.

**Comment by nekrut** [comment]
@nsoranzo - you guys are too productive. let me compare and see

**Comment by nekrut** [comment]
Closed. Instead see https://github.com/galaxyproject/tools-iuc/pull/245

**Pull request closed without merging**